classes/class-wc-connect-jetpack.php

```diff
@@ -142,14 +104,12 @@ public static function build_connect_url( $redirect_url ) {
 		 * @param
 		 */
 		public static function tracks_record_event( $user, $event_type, $data ) {
-			if ( version_compare( JETPACK__VERSION, '7.5', '<' ) ) {
-				if ( function_exists( 'jetpack_tracks_record_event' ) ) {
-					return jetpack_tracks_record_event( $user, $event_type, $data );
-				}
-			} elseif ( class_exists( 'Automattic\\Jetpack\\Tracking' ) ) {
+			if ( class_exists( 'Automattic\\Jetpack\\Tracking' ) ) {
```

Comment:
Nitpick: `jetpack-tracking` is a dependency of `jetpack-connection`, so it will always be available. However, since it's being used explicitly, it should be required explicitly too (both in `composer.json` and in the `Jetpack\Config` initialization).

Reply:
I agree, thanks for pointing it out! Done in 6f4d648f